test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1717,38 +1717,38 @@ static Object createLargeObject(Object arg) throws Throwable {
 		return makeLargeObjectValue.invokeWithArguments(args);
 	}
 
-	static Object createAssorted(MethodHandle makeMethod, String[] fields) throws Throwable {
+	static Object createAssorted(MethodHandle makeMethod, String[] fields, boolean useDefaultValue) throws Throwable {
```

Comment:
The changes to the createAssorted function will also be necessary when creating the suite to test the ACMP bytecode instruction

Reply:
There is a more general way to do that. Also the method should be overload so that the cases that dont require the extra param will not be modified